File: src/main/java/org/jdc/template/domain/main/household/HouseholdManager.java
/*
 * HouseholdManager.java
 *
 * Generated on: 10/14/2013 12:14:38
 *
 */



package org.jdc.template.domain.main.household;


import org.jdc.template.domain.DatabaseManager;

import javax.inject.Inject;

@javax.inject.Singleton
public class HouseholdManager extends HouseholdBaseManager {

    @Inject
    public HouseholdManager(DatabaseManager databaseManager) {
        super(databaseManager);
    }

}

File: src/main/kotlin/org/jdc/template/domain/main/household/HouseholdManager.kt
/*
 * HouseholdManager.kt
 *
 * Generated on: 02/14/2016 07:02:10
 *
 */



package org.jdc.template.domain.main.household

import org.jdc.template.domain.DatabaseManager


@javax.inject.Singleton
class HouseholdManager : HouseholdBaseManager {


    @javax.inject.Inject
    constructor(databaseManager: DatabaseManager): super(databaseManager) {
    }


}